### DISCUSSION TITLE
Help us improve search

### DISCUSSION THREAD

**Discussion by prashantvc** [microsoft/vsmarketplace#411] [Announcement]
We at VS Marketplace started the work to improve the search relevancy.
As part of this effort, we need your help to curate the search results to define baseline KPI

## How can you help?

1. Visit https://extensions.ms/
2. Search for an extension (any of your favourite extensions)
3. Observe the search results, do they make sense?
4. Rearrange results, you can drag and drop, any item need to be on the top? Ranked lower
5. Fill in the feedback, rational behind the rearranging the results
6. Hit submit and repeat for different search terms

**Comment by ssbarnea** [comment]
One suggestion that I have is to find a way to remove outdated extensions from default search results. Allow people to flag extensions as outdated/unmaintained/broken/abandonware, so when they will be excluded from normal search results.

How many of the extensions that were published 4, 6 years ago still do work with current versions of vscode? I bet less than 10%. Even from those that might work they were likely rendered obsolete by newer and better maintained alternatives.

Do not assume that the original publishers will deprecate them. That is a big mistake to do. Once unmaintained to no assume that original OP is still available to make another release or even to unpublish it. That is why we need a deprecation process that does not involve any action from original publisher... In most cases there is nobody to be contacted.

Flagging and reacting on flags is more likely to work.

**Reply by prashantvc** [reply]
Thanks a lot for the feedback!

> One suggestion that I have is to find a way to remove outdated extensions from default search results. 

This on our backlog for the first iteration of search improvements, we are not going to remove the outdated extensions from Marketplace (someone may depend on them), but we will rank them low on the relevance. 

I would love to hear more feedback from you, and chat :)

**Comment by seaniyer** [comment]
Closing out an old announcement
